README.md

```diff
@@ -43,6 +43,14 @@ Or, for the `mime/lite` version:
     mimelite.getType(...); // (Note `mimelite` here)
     <script>
 
+## Deno
+Despite its name this module can be used not just in node context, but also with [deno](https://deno.land), which is kind of the successor of node.js.
+
+    import Mime from "../node-mime/dist/mime.esm.js";
```

Comment:
This import path seems... weird.  What directory is it relative to?  How were these files installed?  In JS/NPM land, these files install into `node_modules/mime`, so I would not expect to see "node-mime" as part of the import path.

This is where my lack of Deno experience raises a lot of questions about how Deno users would normally expect to approach this module.

Reply:
deno would actually refer to modules by URL, something with raw.github...

ill fix it.